src/resolve/test-dnssec-complex.c

```diff
@@ -27,10 +27,11 @@
 #include "bus-common-errors.h"
 #include "dns-type.h"
 #include "random-util.h"
+#include "resolved-def.h"
 #include "string-util.h"
 #include "time-util.h"
 
-#define DNS_CALL_TIMEOUT_USEC (45*USEC_PER_SEC)
+#define DNS_CALL_TIMEOUT_USEC SD_RESOLVED_QUERY_TIMEOUT_USEC
```

Comment:
please drop DNS_CALL_TIMEOUT_USEC complete, and just use the new macro where we used it before